### PR TITLE
add cordova documentation

### PR DESCRIPTION
**Pull request by enguerran**
It refers to some external documentation as the stack is non obvious:

1. cordova
1. java
1. android sdk
1. xcode

etc.

### CODE REVIEW

**Comment by enguerran** [comment]
If you merge #14 first, this PR will have only one commit. If you don't then this PR will add the commit in #14 unless I remove it.

**Comment by enguerran** [comment]
If @y-lohse or @kosssi or someone else want to review [the mobile documentation located here](https://github.com/cozy/cozy-docs-v3/pull/15/files#diff-55c761600825b41a2c6209715bcd4b67), please, be my guest.

File: src/dev/cordova.md [hunk 1]
```diff
@@ -0,0 +1,110 @@
...
+See cordova, android and ios documentation to customize your development environment for your special needs.
+
+Once everything is right, you could run `cordova build` and `cordova run android` to create an APK and push the APK on a device.
+
+__Note:__ The device should be connected with usb.
+See [official android documentation for more details](https://developer.android.com/studio/run/device.html).
+
+### iOS development
+
+For building an iOS app, you need [xcode](https://developer.apple.com/xcode/) and to use xcode, you need [macOS](https://www.apple.com/macos) and to use macOS you need a [mac machine](https://www.apple.com/mac/).
```
Comment (y-lohse) [inline src/dev/cordova.md]: Cheeky. I don't think it's worth mentioning that you need macOS and a mac :)

File: src/dev/cordova.md [hunk 2]
```diff
@@ -0,0 +1,110 @@
+# How to create a mobile application with Cordova
+
+## Install and setup cordova
+
+[Cordova is a tool](http://blog.ionic.io/what-is-cordova-phonegap/) to build apk (Android) and iOS application from web pages.
```
Comment (y-lohse) [inline src/dev/cordova.md]: How about "to build Android and iOS applications"? No need to put the file format first I think.

File: src/dev/cordova.md [hunk 3]
```diff
@@ -0,0 +1,110 @@
...
+
+It works with a CLI that needs [node](https://nodejs.org/en/download/).
+Look at the [cordova documentation to install everything needed](https://cordova.apache.org/docs/en/latest/guide/cli/index.html#installing-the-cordova-cli).
+
+Once cordova is installed, just run `cordova create cozy-app com.example.cozyapp CozyApp` and you got:
```
Comment (y-lohse) [inline src/dev/cordova.md]: I suggest : "and you get the following structure:"

File: src/dev/cordova.md (hunk 1 above)
Comment (enguerran) [reply src/dev/cordova.md]: Pour moi c'est militant. Cool de faire des apps multiplateformes... mais vous ne pourrez pas le faire car il faut un mac pour faire une app pour un iphone. Je trouve que c'est pas si évident et que c'est intéressant de le dénoncer.

**Pull request merged**